application/src/main/java/org/thingsboard/server/config/ThingsboardSecurityConfiguration.java

```diff
@@ -228,7 +234,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .addFilterBefore(buildRestPublicLoginProcessingFilter(), UsernamePasswordAuthenticationFilter.class)
                 .addFilterBefore(buildJwtTokenAuthenticationProcessingFilter(), UsernamePasswordAuthenticationFilter.class)
                 .addFilterBefore(buildRefreshTokenProcessingFilter(), UsernamePasswordAuthenticationFilter.class)
-                .addFilterBefore(requestSizeFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
Will it be the first in the list of all filters?

Reply:
I have added Order(0) to run this filter first

Reply:
Have you checked that it helped? We register filters manually, so I doubt

Reply:
yes, I have checked